token-setup.mkd

```diff
@@ -200,7 +200,7 @@ Logout/login (Is this the only way of doing it?)
 #### Enable Touch policies (Encouraged)
 A touch policy takes advantage of the fact that the Yubikey is a HSM that can be restricted by touch activation.
 
-`$ ykman openpgp keys set-touch <AUT|SIG|ENC> ON`
+`$ ykman openpgp keys set-touch <AUT|SIG|ENC|ATT> ON`
```

Comment:
Maybe mention this feature only apply to YubiKey of version 4.3 and 5 <

Reply:
Sure. The command is a no-op on earlier YubiKeys but I added a note.